qa/tasks/cephfs/fuse_mount.py

```diff
@@ -143,9 +144,12 @@ def _list_fuse_conns(self):
 
         self.client_remote.run(args=['sudo', 'modprobe', 'fuse'],
                                check_status=False)
```

Comment:
i think we should remove the `check_status=False` here too @rishabh-d-dave so that we don't try to mount if the module `fuse` addition failed. or do a check with `lsmod | grep module_name`.

Reply:
It sounds like a good suggestion but I let's do this on a different PR.